.github/workflows/main.yml

```diff
@@ -92,17 +93,20 @@ jobs:
   test-linux-source-distribution:
     needs: create-source-distribution
     runs-on: ubuntu-latest
+    # Build & test job should be run on:
+    # - All Ubuntu LTS supported by Zivid
```

Comment:
```suggestion
    # - All Ubuntu versions supported by Zivid SDK
```

Technically the same, but more robust formulation without the LTS.

Reply:
This is a minor comment. Approving now, fix before merging is you agree.

Reply:
Done